packages/codemirror/src/editor.ts

```diff
@@ -1425,6 +1430,7 @@ namespace Private {
     config: CodeMirrorEditor.IConfig
   ): void {
     let el = editor.getWrapperElement();
+    console.log('losing my mind', option);
```

Comment:
I assume this `log` should be removed as well, right?

Reply:
durn

Reply:
Ha, we've all done worse.  😂

Reply:
ain't that the truth